ecs-cli/modules/commands/usage/usage.go

```diff
@@ -0,0 +1,103 @@
+	ComposeCreate = "Creates an ECS task definition from your compose file. Note that we do not recommend using plain text environment variables for sensitive information, such as credential data."
+	ComposePs     = "Lists all the containers in your cluster that were started by the compose project."
+	ComposeUp     = "Creates an ECS task definition from your compose file (if it does not already exist) and runs one instance of that task on your cluster (a combination of create and start)."
+	ComposeStart  = "Starts a single task from the task definition created from your compose file."
+	ComposeRun    = "Starts all containers overriding commands with the supplied one-off commands for the containers."
+	ComposeStop   = "Stops all the running tasks created by the compose project."
+	ComposeScale  = "Scales the number of running tasks to the specified count."
+)
+
+// Compose Service
+const (
+	Service       = "Manage Amazon ECS services with docker-compose-style commands on an ECS cluster."
+	ServiceCreate = "Creates an ECS service from your compose file. The service is created with a desired count of 0, so no containers are started by this command. Note that we do not recommend using plain text environment variables for sensitive information, such as credential data."
+	ServiceStart  = "Starts one copy of each of the containers on an existing ECS service by setting the desired count to 1 (only if the current desired count is 0)."
+	ServiceUp     = "Creates a new ECS service or updates an existing one according to your compose file. For new services or existing services with a current desired count of 0, the desired count for the service is set to 1. For existing services with non-zero desired counts, a new task definition is created to reflect any changes to the compose file and the service is updated to use that task definition. In this case, the desired count does not change."
+	ServicePs     = "Lists all the containers in your cluster that belong to the service created with the compose project."
+	ServiceScale  = "Scales the desired count of the service to the specified count."
+	ServiceStop   = "Stops the running tasks that belong to the service created with the compose project. This command updates the desired count of the service to 0."
+	ServiceRm     = "Updates the desired count of the service to 0 and then deletes the service."
+)
+
+// Configure
+const (
+	Configure               = "Stores a single cluster configuration."
+	ConfigureDefault        = "Sets the default cluster config."
+	ConfigureMigrate        = "Migrates a legacy ECS CLI configuration file to the current YAML format."
+	ConfigureProfile        = "Stores a single profile."
+	ConfigureProfileDefault = "Sets the default profile."
+)
+
+// Image
+const (
+	Push   = "Pushes an image to an Amazon ECR repository."
+	Pull   = "Pulls an image from an Amazon ECR repository."
+	Images = "Lists images an Amazon ECR repository."
```

Comment:
suggested correction: "Lists images from an Amazon ECR repository. Lists all images in all repositories by default."